services/src/pom.xml

```diff
@@ -69,6 +69,7 @@
         <validation-api.version>2.0.0.Final</validation-api.version>
         <hibernate-validator.version>6.0.10.Final</hibernate-validator.version>
         <glassfish-el.version>3.0.1-b09</glassfish-el.version>
+        <com.auth0.version>3.2.0</com.auth0.version>
```

Comment:
Why do you specify the version for the group instead of the artifact?

Reply:
Because 'java-jwt' is the only one artifact which is used with this group. And also it was made just by analogy with other version properties. Most of them has group version.

Reply:
I renamed it